storage/src/mmr/journaled.rs

```diff
@@ -1,32 +1,57 @@
 //! An MMR backed by a fixed-item-length journal.
+//!
+//! A [`crate::journal`] is used to store all unpruned MMR nodes, and a [`crate::metadata`] store is
```

Comment:
In other places, we don't use backticks (`[crate::journal]`). Not sure what is more correct 🤔

Reply:
removed the backticks.